ajax/libs/image-map-resizer/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "image-map-resizer",
+  "version": "1.0.3",
+  "filename": "js/imageMapResizer.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/davidjbradshaw/image-map-resizer.git"
+  },
+  "description": "Make the HTML image map responsive, so the map scales with the image",
+  "keywords": [
+    "imagemap",
+    "map",
+    "image",
+    "Resizing",
+    "Resizer",
+    "resize",
+    "RWD",
+    "responsive",
+    "jquery-plugin"
+  ],
+  "license": "MIT",
+  "npmName": "image-map-resizer",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(js|css)/**/imageMapResizer*"
```

Comment:
Since there is different naming in old versions, e.g. jquery.imageMapResizer.js in [v0.1.3]( https://github.com/davidjbradshaw/image-map-resizer/tree/v0.1.3/js), let's use ` "+(js|css)/**/*imageMapResizer*"`